indico/modules/designer/pdf.py

```diff
@@ -97,6 +97,8 @@ def _draw_item(self, canvas, item, tpl_data, content, margin_x, margin_y):
         style = ParagraphStyle({})
         style.alignment = ALIGNMENTS[item['text_align']]
         style.textColor = COLORS[item['color']]
+        style.backColor = item['background_color'] if item['background_color'] else None
```

Comment:
I don't think we have a `background_color` for older badges, do we?

```suggestion
        style.backColor = item.get('background_color') or None
```

the ` or None` can be omitted if you do not need to normalize other falsy values such as empty strings to None

Reply:
That right, there is no `background_color` for older badges